### PR TITLE
Delete CUndefined

### PR DESCRIPTION
**Pull request by djspiewak**
Breaking because I also renamed `fromJValueRaw` to `unsafeFromJValueRaw`.

### CODE REVIEW

**Comment by djspiewak** [comment]
[ch163]

**Comment by djspiewak** [comment]
I'm starting to think that hoarder just doesn't work…

**Comment by alissapajer** [comment]
@djspiewak do you want to WIP this until it's working?

**Comment by djspiewak** [comment]
@alissapajer I'm not going to rebase it, so I'll just leave it as-is for the time being. If it drags on for a while, I'll just close the PR.

File: blueeyes/src/main/scala/quasar/precog/common/CValue.scala [hunk 1]
```diff
@@ -178,7 +173,15 @@ object RValue extends RValueInstances {
...
 
-  def fromJValueRaw(jv: JValue): RValue = {
+  /**
+   * This function is unsafe and should be deprecated! Always prefer `fromJValue`.
+   *
+   * NB: This function *does* serve a mildly useful purpose, which is converting from
+   * JValue to RValue without numeric fitting (note the use of toCValueRaw). Just
+   * the fact that it doesn't return Option[RValue] causes the unsafety. Ultimately,
+   * we should remove the unsafety from the function while keeping the raw semantics.
+   */
```
Comment (edmundnoble) [inline blueeyes/src/main/scala/quasar/precog/common/CValue.scala]: :+1:

**Comment by djspiewak** [comment]
I'm not going to be able to finish this before 0.10. Closing for now.

**Pull request closed without merging**